### PR TITLE
Clarify PS command to find cloud-only users

### PR DESCRIPTION
**Pull request by gh-andrem**
The command found users where DirSyncEnabled flag is not set to $false which means you can find users with DirSyncEnabled flag set to true and therefore only find on-prem users.

I also rearanged the paragraphs to make it more clearer what each command is for.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/enterprise/view-user-accounts-with-microsoft-365-powershell.md [hunk 1]
```diff
@@ -104,17 +104,17 @@ User accounts have two sources:
 
 - Azure Active Directory (Azure AD) AD accounts, which are created directly in the cloud.
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/view-user-accounts-with-microsoft-365-powershell.md]:
```suggestion
- Azure Active Directory (Azure AD) accounts, which are created directly in the cloud.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @amueller-tf, kindly commit the minor additional suggested change. Many thanks.

**Comment by gh-andrem** [comment]
@JohanFreelancer9 Thanks for double-checking. I just made the suggested changes.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**